docs/document/docs/guides/format-setting-plugin.md

```diff
@@ -57,6 +57,37 @@ src/
 ├── manifest.json
 ```
 
+## Create a Plug-in Icon
+- The icon image for the Plug-in.
+- This file is required.
+- Use the following file types: png, jpg, gif, bmp
+- The maximum file size is 20MB.
+- Please add icon file into folder ``image/icon.png``
```

Comment:
I think you don't need to write precise requirements for icon.
You can link the detail using the official guide.
Here is my suggestion.

```suggestion
## Create a plug-in icon
Create a plug-in icon following the [guide](https://kintone.dev/en/plugins/introduction-to-plug-ins/steps-for-plug-in-development/#creating-each-file).<br>
Place the icon file (icon.png) in the image folder.
```